src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -571,7 +571,7 @@ void shouldNotDisplaySeriesPosition_whenBookDoesNotHaveSeriesPosition() {
     }
 
     @Test
-    void shouldAddBookToDatabaseWhenSaveEventIsCalled() {
+    void addBookToDatabaseWhenSaveEventIsCalled() {
```

Comment:
Could you please rename this to `addBookToDatabaseWhenSaveEventFired` (I appreciate this wasn't there before)?